keadm/cmd/keadm/app/cmd/util/rpminstaller.go

```diff
@@ -51,13 +56,23 @@ func (r *RpmOS) InstallMQTT() error {
 		return nil
 	}
 
-	// install MQTT
-	for _, command := range []string{
-		"yum -y install epel-release",
+	commands := []string{
 		"yum -y install mosquitto",
 		"systemctl start mosquitto",
 		"systemctl enable mosquitto",
-	} {
+	}
+
+	if vendor, err := getOSVendor(); err != nil {
+		return err
+	} else {
+		// epel-release package does not included in openEuler OS
+		if vendor != openEulerOS {
+			commands = append(commands[:1], "yum -y install epel-release")
```

Comment:
It seems that the commands are executed in sequence , so it should be like this `commands = append([]string{"yum -y install epel-release"}, commands...)`